ITEM.tsv

```diff
@@ -5553,20 +5553,20 @@ ITEM_20150717_005552	There is not much like this to have as your guard.
 ITEM_20150717_005553	If you take a look at this staff, you can see a great transformation has simplified it. This does not mean the power has been greatly reduced.
 ITEM_20150717_005554	- All Cryomancer Skill Levels increased by +1
 ITEM_20150717_005555	- Earthquake Skill Levels increased by +3
-ITEM_20150717_005556	
-ITEM_20150717_005557	
+ITEM_20150717_005556    The catastrophic effect of this equipment is activated independently from the user's skill.
+ITEM_20150717_005557    From all the people that have used this equipment, we are rooting for you to become the most renowned throughout history.	
 ITEM_20150717_005558	- Basic Attacks have 1% chance to decrease Magic Defense and increase Physical Defense for 30 seconds
 ITEM_20150717_005559	- Basic Attacks have 2% chance to ignite the target for 10 seconds
-ITEM_20150717_005560	
-ITEM_20150717_005561	
-ITEM_20150717_005562	
-ITEM_20150717_005563	
-ITEM_20150717_005564	
-ITEM_20150717_005565	
-ITEM_20150717_005566	
-ITEM_20150717_005567	
-ITEM_20150717_005568	
-ITEM_20150717_005569	
+ITEM_20150717_005560    We have found death without discretion. It is time to apply what we have discovered on the enemy.	
+ITEM_20150717_005561    You can't always draw power from wandering souls. However, if you succeed. An item that which increases your hidden potential will be created. 	
```

Comment:
"However, if you succeed an item that which increases your hidden potential will be created."